templates/includes/sidebar.html

```diff
@@ -1,4 +1,15 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <li>Username: <a href="{% url 'taxi:driver-detail' user.id %}">{{ user.username }}</a> </li>
```

Comment:
Ensure that the `user.id` used in the URL for 'taxi:driver-detail' corresponds to a `Driver` instance. This is important to avoid incorrect URL generation and potential runtime errors.